elo_boat.py

```diff
@@ -47,7 +47,7 @@
 _guild = None
 baseurl = "https://api.wc3stats.com"
 upload_channel = None
-NO_POWER_MSG = "You do not have enough power to perform such an action."
+NO_POWER_MSG = "You do not have permission to use this command."

```

Comment:
This isn't intended to be used for command only, it might be enlarged in the future on generalized events. Discords commands aren't the only way to interract with discord bot. You can formulate it differently if you want : 
-"You do not have permission to perform such an action"
-"You can't do that"

If you want specific response, you need to create a handler that retrieve the type of the discord event and filter by commands => rest is then generic message

Reply:
For instance i intent to transform the captains mode to a more evolved and user friendly version of it like what is done with OKIB bot